core/discovery/event.go

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package discovery
+
+// Topic represents the different topics a consumer can subscribe to
+const (
+	// ProposalAddedEventTopic represents newly announced proposal
+	ProposalAddedEventTopic = "ProposalAdded"
```

Comment:
Go's naming convention for multiple constants is to to prefix. See example https://github.com/golang/go/blob/c04e47f82159f1010d7403276b3dff5ab836fd00/src/net/http/status.go#L9

```go
const (
    EventTopicProposalAdded = "ProposalAdded"
    EventTopicProposalUpdated = "ProposalUpdated"
    EventTopicProposalRemovedE = "ProposalRemoved"
)
```

When it is more easy to find all topics when writing in IDE `discovery.EventTopic*`

Reply:
Named by existing event convention. Renamed